docs/DeveloperGuide.md

```diff
@@ -1,14 +1,123 @@
-# Developer Guide
+# Table of Content
+
+- [Acknowledgements](#acknowledgements)
+- [Design and Implementation](#design-and-implementation)
+  - [Architecture](#architecture)
+  - [Parser](#parser)
+  - [Update feature](#update-feature)
+  - [EquipmentManager](#equipmentmanager)
+- [Product scope](#product-scope)
+  - [Target user profile](#target-user-profile)
+  - [Value proposition](#value-proposition)
+- [User stories](#user-stories)
+- [NonFunctional requirements](#nonfunctional-requirements)
+- [Glossary](#glossary)
+- [Instructions for manual testing](#instructions-for-manual-testing)
 
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
You could maybe remove this placeholder in the final version?

Reply:
Noted, will do so!